### PR TITLE
[1.10] Add `setScriptExecution` page method

### PR DESCRIPTION
**Pull request by osbre**
Same as https://github.com/puppeteer/puppeteer/blob/c4bad4a6dade2eafa6a8145077a1103153467e8e/packages/puppeteer-core/src/common/EmulationManager.ts#L212-L220

### CODE REVIEW

**Comment by enricodias** [comment]
Thanks for your contribution. I think it's a useful feature, but it's kinda odd to call a function with "enabled" in the name to disable something. How about just `setScriptExecution(bool $enabled)`?

**Comment by osbre** [comment]
Thanks, @enricodias. Updated the name.

File: src/Page.php [hunk 1]
```diff
@@ -1068,4 +1068,25 @@ public function setUserAgent(string $userAgent)
...
+    /**
+     * Disable or enable JavaScript execution for the current page.
+     *
+     * @param bool $enabled
+     *
+     * @throws CommunicationException
+     *
+     * @return ResponseWaiter
+     */
+    public function setScriptExecution(bool $enabled)
```
Comment (enricodias) [inline src/Page.php]:
```suggestion
     * @throws CommunicationException
     */
    public function setScriptExecution(bool $enabled): ResponseWaiter
```

Those param and return docblocks are unnecessary if we use the types correctly directly in the method itself. Other then that, the code looks good to me.

**Comment by GrahamCampbell** [comment]
Note that we follow semver, so this needs to target the 1.10 branch.

**Review by enricodias — APPROVED** [review]

**Comment by GrahamCampbell** [comment]
Thanks.

**Pull request merged**